content/tutorials/pulumi-deployments-click-to-deploy/index.md

```diff
@@ -0,0 +1,120 @@
+---
+title_tag: Create and Manage cloud infrastructure with Pulumi Deployments | Learn Pulumi
+title: "Create Infrastructure with Pulumi Deployments Click-to-Deploy"
+layout: single
+description: |
+    Create infrastructure deployments using the New Project Wizard and GitHub integration to enable push-to-deploy in the Pulumi Cloud.
+meta_desc: In this step-by-step tutorial, learn how to create and manage cloud infrastructure using Pulumi Deployments, the New Project Wizard, and GitHub integration.
+meta_image: meta.png
+weight: 50
+summary: |
+    In this tutorial, you'll learn how to create, deploy, and manage cloud infrastructure using Pulumi Deployments with Click-to-Deploy. You'll start by using the New Project Wizard to set up and launch infrastructure from your browser, selecting a template to get started quickly. Additionally, you'll configure the Pulumi GitHub App to support additional deployment triggers, including push-to-deploy.
+you'll_learn:
+  - How to use the New Project Wizard to create and deploy a Pulumi project from your browser
+  - How to install and configure the Pulumi GitHub App
+  - How to trigger and manage deployments directly from the Pulumi Cloud console using click-to-deploy
+prereqs:
+    - Completion of the [Getting Started](/docs/get-started/) guide or familiarity with the basics of the Pulumi workflow.
+    - The [Pulumi CLI](/docs/install/)
+    - A [Pulumi Cloud account](https://app.pulumi.com/signup)
+    - An [Amazon Web Services](https://aws.amazon.com/) account
```

Comment:
Probably worth mentioning you'll need programmatic access and credentials in some form as well. (See my other comment about setting environment variables.)